ajax/libs/evaporate/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "evaporate",
+  "filename": "evaporate.min.js",
+  "version": "2.0.4",
+  "description": "Javascript library for browser to S3 multipart resumable uploads",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/TTLabs/EvaporateJS.git"
+  },
+  "keywords": [
+    "aws",
+    "amazon",
+    "S3",
+    "multipart",
+    "multi-part",
+    "uploads",
+    "browser",
+    "transfer",
+    "acceleration",
+    "signature version 4",
+    "v4"
+  ],
+  "author": {
+    "name": "Tom Saffell",
+    "email": "tcs@videopixie.com"
+  },
+  "license": "BSD 3-Clause",
```

Comment:
This was the commit that removed the `-` because it was reportedly not compatible.

https://github.com/TTLabs/EvaporateJS/commit/9c17656193190f993982d18ef1757184fa97ecd7

So I did not re-add it to the corrected BSD 3 name.